make.ps1

```diff
@@ -157,6 +157,7 @@ elseif ($command -eq "check")
 }
 elseif ($command -eq "docs")
 {
+	./make.ps1 version
```

Comment:
This will set the version string twice now:
* https://ci.appveyor.com/project/OpenRA/openra/build/1.0.2248#L139
* https://ci.appveyor.com/project/OpenRA/openra/build/1.0.2248#L156